pkg/mqtrigger/scalermanager.go

```diff
@@ -0,0 +1,548 @@
+package mqtrigger
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	fv1 "github.com/fission/fission/pkg/apis/core/v1"
+	"github.com/fission/fission/pkg/crd"
+	"github.com/fission/fission/pkg/utils"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	k8sCache "k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	scaledObjectGVR = schema.GroupVersionResource{
+		Group:    "keda.k8s.io",
+		Version:  "v1alpha1",
+		Resource: "scaledobjects",
+	}
+	authTriggerGVR = schema.GroupVersionResource{
+		Group:    "keda.k8s.io",
+		Version:  "v1alpha1",
+		Resource: "triggerauthentications",
+	}
+	matchFirstCap = regexp.MustCompile("(.)([A-Z][a-z]+)")
+	matchAllCap   = regexp.MustCompile("([a-z0-9])([A-Z])")
+)
+
+func getDynamicClient() (dynamic.Interface, error) {
```

Comment:
Can this be moved to a place where we get other clients - as this is reusable across Fission

Reply:
Done